package.json

```diff
@@ -166,6 +167,10 @@
     "request": "2.88.0",
     "rimraf": "2.6.2",
     "rocambole": "0.7.0",
+    "rollup": "^0.66.6",
```

Comment:
Likely want to pin these dependencies and let renovate update them.

Which means you'll need some tests to ensure updates don't break anything!